lab1.js

```diff
@@ -134,7 +158,21 @@ var tooHungryDay;
 */
 
 // your code goes here
+var meals = 0;
+tooHungryDay = -1;
+
+for (var i = 0; i < mealsPerDay.length; i++) {
+  meals = meals + mealsPerDay[i];
+  var avg = meals / (i + 1);
+
+  console.log('Day ' + (i + 1) + ' average meals per day that lion got since new caretaker started is ' + avg);
 
+  if (avg < 4 && tooHungryDay < 0) {
```

Comment:
I tried putting break, but the last few console.logs after the if condition was executed were not printing and I thought that it was a requirement to print all of them, so I used this method.
Thanks!
Date: Thu, 16 Jul 2015 23:37:01 -0700
From: notifications@github.com
To: lab1-adv-track@noreply.github.com
CC: tina_2212@hotmail.com
Subject: Re: [lab1-adv-track] add and commit of advanced track lab1 (#1)

In lab1.js:

>  
> +  if (avg < 4 && tooHungryDay < 0) {

good job checking to see if you're already assigned a value to tooHungryDay. You can also use a break at the end of this if block to end execution of the for loop.


—
Reply to this email directly or view it on GitHub.